docs/core/settings/variables.md

```diff
@@ -118,15 +199,13 @@ See also:
 
 Variables that work nearly everywhere where there is a username:
 
-| Variable | Long Name | Description |
-| -------- | --------- | ----------- |
-| `%u` | `user` | full username (e.g. user@domain) |
-| `%n` | `username` | user part in user@domain, same as `%u` if there's no domain |
-| `%d` | `domain` | domain part in user@domain, empty if user with no domain |
-| | `session` | session ID for this client connection (unique for 9 years) |
-| | `auth_user` | SASL authentication ID (e.g. if master user login is done, this contains the master username). If username changes during authentication, this value contains the original username. Otherwise the same as `%{user}`. |
-| | `auth_username` | user part in `%{auth_user}` |
-| | `auth_domain` | domain part in `%{auth_user}` |
+| Variable | Description |
+| -------- | ----------- |
+| `%{user}` | Full username (e.g. user@domain) |
+| `%{user | username}` | User part in user@domain, same as `%{user}` if there's no domain |
+| `%{user | domain}` | Domain part in user@domain, empty if user with no domain |
+| `session` | Session ID for this client connection (unique for 9 years) |
```

Comment:
These %{..} are now used inconsistently between tables. Some have them, some don't. Should stay consistent. In this table it's inconsistent even within the table.